tests/cases/fourslash/organizeImportsAttributes3.ts

```diff
@@ -0,0 +1,20 @@
+/// <reference path="fourslash.ts" />
+
+////import { A } from "./a";
+////import { C } from "./a" assert {      type: "a" };
+////import { Z } from "./z";
+////import { A as D } from "./a" assert    { type: "b" };
+////import { E } from "./a" assert { type: /* comment*/ "a"              };
+////import { F } from "./a" assert     {type: "a" };
+////import { Y } from "./a"   assert{ type: "b" /* comment*/};
```

Comment:
You could add another test showing that

```ts
import { A } from "./a" assert { foo: "foo", bar: "bar" };
import { B } from "./a" assert { bar: "bar", foo: "foo" };
```

can be combined since you sorted the elements.

Reply:
Nice idea!